cosmosdb/create-cosmosdb-gremlin-account/create-cosmosdb-gremlin-account.sh

```diff
@@ -32,11 +37,11 @@ az cosmosdb database create \
 	--resource-group $resourceGroupName
 
 
-# Create a graph with a partition key and 1000 RU/s
+# Create a graph with a partition key and 400 RU/s
 az cosmosdb collection create \
 	--collection-name $graphName \
 	--name $accountName \
 	--db-name $databaseName \
 	--resource-group $resourceGroupName \
-    --partition-key-path /mypartitionkey \
-    --throughput 1000
\ No newline at end of file
+    --partition-key-path $partitionKeyPath \
+    --throughput throughput
```

Comment:
Causes an error - should be `$throughput`?

Reply:
Thanks for catching. Fixed in next commit.